sdcm/stress_thread.py

```diff
@@ -17,6 +17,8 @@
 import time
 import logging
 import concurrent.futures
+import itertools
+from string import Template
```

Comment:
why did you use Template instead of simple string formatting?

Reply:
cause it's a complex yaml file, which already has {} in the comment for the stress commands